R/dsl.R

```diff
@@ -7,3 +7,29 @@
 uiicon <- function(type = "", ...) {
   tags$i(class = paste(type, "icon"), ...)
 }
+
+#' This creates a default dropdown using Semantic UI styles with Shiny input. Dropdown is already initialized
+#' and available under input[[name]].
+#'
+#' @param name Input name. Reactive value is available under input[[name]].
+#' @param choices All available options one can select from.
+#' @param choices_value What reactive value should be used for corresponding choice.
+#' @param default_text Text to be visible on dropdown when nothing is selected.
+#' @param value Pass value if you want to set selection for dropdown.
```

Comment:
I would write sth like that `@param value Pass value if you want to initialize selection for dropdown default NULL`